galaxy/main/migrations/0090_auto_20180411_1129.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you please give more verbose names to migration files.

Reply:
Done